### DISCUSSION TITLE
Wifi Drivers

### DISCUSSION THREAD

**Discussion by TheKurgan13** [elementary/triage#654] [Q&A] [answered]
I’m downloading Elementary OS on my 2014 Mac and need to ensure I have the correct Wi-Fi drivers for internet connectivity. Does Elementary OS 8 include all necessary drivers for my laptop, or will I need to install additional ones for older hardware? If additional drivers are required, where can I find them on the Elementary website? Thank you.

**Comment by teamcons** [comment, marked as answer]
elementary OS depends for anything lower level, on Ubuntu, since it is built on top of it
so you may need to look up "Ubuntu 24.04" + your model for any potential issue

there is no driver on the elementary website.
In system settings there is a driver manager that can offer some

**Reply by TheKurgan13** [reply]
Thank you for your response. I installed a preview of Elementary OS and found the driver you mentioned. However, when I attempted to wipe and install the OS, I couldn’t select a drive for installation; the options were missing, and the “Next” button was grayed out.
